src/lib.rs

```diff
@@ -124,6 +124,9 @@ use std::boxed::Box;
 #[cfg(not(feature = "std"))]
 use alloc::boxed::Box;
 
+/// Error type that allows custom implementations of the Serialize trait.
+pub use ser::Error;
```

Comment:
That's true but wouldn't it be better to then use `use slog::ser::*;` and move `Serialize` and `Record` into the `ser` module?